src/pages/what-we-do.pug

```diff
@@ -20,30 +20,30 @@ block pageComponents
         blockImage: {
           src: "/assets/images/design.jpg",
-          alt: ""
+          alt: "Writing our recommendations"
         },
         blockModifer: "reverse",
       })
 
       +block({
-        blockHeading: "Development",
-        blockContent: "<p>We use technologies like <strong>React</strong>, <strong>Node.js</strong> and <strong>Ruby</strong> to build and deliver the designed solutions. To verify success, we measure our results with state of the art analytics.</p><p>Everything we build is backed by extensive <strong>automated testing</strong>, stringent <strong>code review</strong> processes, and live <strong>performance monitoring</strong> to ensure continual bug-free operation.</p>",
+        blockHeading: "3. Implementation",
+        blockContent: "<p>With a series of recommendations in hand, our team of software engineers and designers begin to implement them in priority order to ensure value is delivered early. Typically, this involves work like:</p><ul><li><strong>Creating or updating web and mobile apps</strong>, so that customers and staff can interact with your business on a self-service basis, from anywhere, and at any time.</li><li><strong>Improving information capture</strong>, so that data is more accurate, operating costs are reduced, and customer frustration is lower.</li><li><strong>Building Application Programming Interfaces (APIs)</strong>, so B2B partners can write software that connects with your business, and you can build new software products and services more easily.</li><li><strong>Integrating your systems with other APIs</strong> to automate common tasks, deliver additional functionality to your customers, and allow you to offer new products and services.</li><li><strong>Automatically enriching and cleaning up customer data</strong>, so that you can operate with high quality, immediately available information.</li><li><strong>Automatically extracting structured data from scans and images</strong>, so that your staff don't need to manually process physical documents or photos of documents.</li><li><strong>Automating business reporting and analytics</strong>, so that you always have the information you need to hand, and no staff time is being wasted manually collating this for you.</li></ul><p>We'll keep you involved at every step of the implementation, with sensitity to existing procedures, any staff training required, and other business deadlines. This way, you can relax knowing that delivery is going to be as frictionless as possible.</p>",
```

Comment:
- We have a styled list for this exact use case.
- I think both the intro and the outro paragraphs can be condensed.
- We can apply a loose '2-line' rule to list items with some minor trimming of content.

> <p>With sensitivity to existing business procedures and deadlines, our team of software engineers and designers begin to implement our recommendations in priority order to ensure value is delivered early. Typically, this involves:</p><ul class="styled-list styled-list--inside-sm"> <li class="styled-list__item"><strong>Creating or updating web and mobile apps</strong>, so that customers and staff can interact with your business on a self-service basis, from anywhere, and at any time.</li><li class="styled-list__item"><strong>Improving information capture</strong>, so that data is more accurate, operating costs are reduced, and customer frustration is lower.</li><li class="styled-list__item"><strong>Building APIs</strong>, so B2B partners can write software that connects with your business, and you can build new software products and services more easily.</li><li class="styled-list__item"><strong>Integrating third-party APIs</strong> to automate common tasks, deliver additional functionality to your customers, and allow you to offer new products and services.</li><li class="styled-list__item"><strong>Automatically enriching and cleaning up customer data</strong>, so that you can operate with high quality, immediately available information.</li><li class="styled-list__item"><strong>Automatically extracting structured data from scans and images</strong>, so that your staff don't need to manually process physical documents or photos of documents.</li><li class="styled-list__item"><strong>Automating reporting and analytics</strong>, so that you always have the information you need to hand, and no staff time is being wasted manually collating this for you.</li></ul>

`<p>With sensitivity to existing business procedures and deadlines, our team of software engineers and designers begin to implement our recommendations in priority order to ensure value is delivered early. Typically, this involves:</p><ul class="styled-list styled-list--inside-sm"> <li class="styled-list__item"><strong>Creating or updating web and mobile apps</strong>, so that customers and staff can interact with your business on a self-service basis, from anywhere, and at any time.</li><li class="styled-list__item"><strong>Improving information capture</strong>, so that data is more accurate, operating costs are reduced, and customer frustration is lower.</li><li class="styled-list__item"><strong>Building APIs</strong>, so B2B partners can write software that connects with your business, and you can build new software products and services more easily.</li><li class="styled-list__item"><strong>Integrating third-party APIs</strong> to automate common tasks, deliver additional functionality to your customers, and allow you to offer new products and services.</li><li class="styled-list__item"><strong>Automatically enriching and cleaning up customer data</strong>, so that you can operate with high quality, immediately available information.</li><li class="styled-list__item"><strong>Automatically extracting structured data from scans and images</strong>, so that your staff don't need to manually process physical documents or photos of documents.</li><li class="styled-list__item"><strong>Automating reporting and analytics</strong>, so that you always have the information you need to hand, and no staff time is being wasted manually collating this for you.</li></ul>`